kuma/static/styles/components/wiki/quick-links.scss

```diff
@@ -89,26 +89,28 @@
 
     /* sidebar icons - deprecated, obsolete, experiemental, etc */
     .sidebar-icon {
-        @include bidi(
-            (
-                (margin-left, $grid-spacing * -1, 5px),
-                (margin-right, 5px, $grid-spacing * -1)
-            )
-        );
-        opacity: .3;
+        @include bidi((
+            (margin-left, 0, $grid-spacing - $icon-size),
+            (margin-right, $grid-spacing - $icon-size, 0)
+        ));
 
-        /* only the first sidebar-icon gets pulled into the gutter */
-        & + .sidebar-icon {
-            @include bidi(((margin-left, 0, 5px), (margin-right, 5px, 0)));
-        }
+        opacity: .3;
 
         #{$selector-icon} {
             @include set-font-size($base-font-size);
-            min-width: 15px;
+            width: $icon-size;
             vertical-align: top;
         }
     }
 
+    /* only the first sidebar-icon gets pulled into the gutter */
+    .sidebar-icon:first-child {
+        @include bidi((
+            (margin-left, $grid-spacing * -1, $grid-spacing - $icon-size),
+            (margin-right, $grid-spacing - $icon-size, $grid-spacing * -1)
+        ));
```

Comment:
This fixes the issue where if the page title is too long, then the icon will push the `<a>` link one line lower.

This can be seen on&nbsp;the&nbsp;[**CSS&nbsp;Basic User&nbsp;Interface**](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Basic_User_Interface) page:
<details><summary><strong>Before</strong></summary>

![Kuma PR‑5131 (Before)](https://user-images.githubusercontent.com/3889017/49197941-3a605780-f391-11e8-8213-b4bd456461f6.png)

</details><details><summary><strong>After</strong></summary>

![Kuma PR‑5131 (After)](https://user-images.githubusercontent.com/3889017/49197963-52d07200-f391-11e8-9747-5336255efd9b.png)

</details>

Reply:
Interesting use of ``<details><summary>`` in a GitHub comment.

Reply:
I saw it used by [Greenkeeper](https://greenkeeper.io) and in a few other places.